src/parachain/escrow.ts

```diff
@@ -220,12 +225,12 @@ export class DefaultEscrowAPI implements EscrowAPI {
 
         // Note: on first time staking the user has to provide both a increased stake and a locktime
         // otherwise, the rewards will be 0.
-        let newStakedBalance = {
+        const newStakedBalance = {
```

Comment:
Thanks for fixing the linting errors!

Reply:
I'd like to take credit, but my editor did all the work when I saved the file :)